superset/utils/pandas_postprocessing.py

```diff
@@ -577,7 +578,7 @@ def contribution(
     :return: DataFrame with contributions.
     """
     contribution_df = df.copy()
-    numeric_df = contribution_df.select_dtypes(include="number")
+    numeric_df = contribution_df.select_dtypes(include=["number", Decimal])
```

Comment:
Bycatch: fix index errors for Percent metrics when Decimal columns are used in metrics.